### ISSUE TITLE
Video was set to private

### ISSUE THREAD

**Issue by fluorita** [LorenzHW/My-Telegrams#4]
Hi i would like to use this skill i understand that you took it down but could you please set the tutorial video to public again because now idk how to install it.

Thanks in advance :D

**Comment by LorenzHW** [comment]
Hi fluorita.

Thanks for your interest in the skill. Yes, I took the videos down. The videos won't help you to install the skill. However, I am working on a new version of the skill. I'll keep this issue open and notify you once the skill is live or in beta. Thanks.

**Comment by fluorita** [comment]
> Hi fluorita.
> 
> Thanks for your interest in the skill. Yes, I took the videos down. The videos won't help you to install the skill. However, I am working on a new version of the skill. I'll keep this issue open and notify you once the skill is live or in beta. Thanks.

Im so happy thank you a lot.

**Comment by fluorita** [comment]
If you need a beta tester lmk 😅

**Comment by LorenzHW** [comment]
Thanks. Will do :)

**Comment by fluorita** [comment]
I have a question will your skill have any touch input for devices with a screen like the echo show 5?

**Comment by LorenzHW** [comment]
No, personally I don't own an echo show, so I won't implement it. But since this skill will be open source, feel free to add it.

**Comment by fluorita** [comment]
Ok ill try :D when its released btw more or less no pressure just asking 😅 when do you think you'll release the skill?

**Comment by LorenzHW** [comment]
Hmm, can't really tell. Hopefully, by the end of this year :)

**Comment by LorenzHW** [comment]
> If you need a beta tester lmk 😅

Hey, could you give me your E-Mail then I can invite you for the beta. Should happen soon :)
